TWLight/users/views.py

```diff
@@ -87,15 +87,18 @@ def get_object(self):
         get_object in order to be able to use the SelfOnly mixin.
         """
         assert "pk" in list(self.kwargs.keys())
+
         try:
-            return User.objects.get(pk=self.kwargs["pk"])
+            if self.kwargs["pk"] == None :
+                return User.objects.get(self.kwargs["pk"])
         except User.DoesNotExist:
             raise Http404
```

Comment:
Unless I'm missing something, it looks like this will only try `User.objects.get()` when there is no user to get.
Can you tell me what this is doing, or what problem you were trying to clear up with the change?

Reply:
Hello Jason,
I apologize for my mistake.  It was a typo.  
Please let me know if you have any questions during the code review. 
Thank you and stay safe,
Kangyoon

Reply:
@jsnshrmn 
Hey Jason,
Travis CI is failing due to different locales messages.  If you don't mind, could you advise me to resolve this issue?
Thanks,
Kangyoon

Reply:
Your PR's failing on code style issues ([take a look here](https://travis-ci.org/github/WikipediaLibrary/TWLight/builds/697685024#L2629)). Running `docker exec <CONTAINER_ID> /venv/bin/black -t py37 /app/TWLight` and committing the changes should make your PR pass that check.